src/translations/i18n.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+FICHIERS=$(ls -1 src/translations/*.json)
```

Comment:
What is `FICHIERS`? Is that french for `FILES`?

Reply:
oh, yes. 😅